internal/pkg/cli/completion.go

```diff
@@ -67,10 +74,16 @@ The code must be evaluated to provide interactive completion of commands.`,
   Install bash completion on linux
   /code $ source <(copilot completion bash)
   /code $ copilot completion bash > copilot.sh
-  /code $ sudo mv copilot.sh /etc/bash_completion.d/copilot`,
+  /code $ sudo mv copilot.sh /etc/bash_completion.d/copilot
+
+	Install fish completion
```

Comment:
Can we update the spacing for this statement so that it aligns with the other "Install" sentences 🙏

Reply:
Sure, totally.